go.mod

```diff
@@ -3,27 +3,27 @@ module github.com/openshift/kube-compare
 go 1.22.0
 
 require (
-	github.com/BurntSushi/toml v0.3.1
+	github.com/BurntSushi/toml v1.3.2
```

Comment:
I find it troubling that our root-level go.mod has a version bump but there's no accompanying change to `vendor/github.com/BurntSushi/toml` - I would expect we'd need both?

Reply:
Ah, I see - One of the submodluse must have had the newer version, so the vendor/ area already had it...